daemon/graphdriver/vfs/driver.go

```diff
@@ -75,6 +99,16 @@ func (d *Driver) Remove(id string) error {
 
 func (d *Driver) Get(id, mountLabel string) (string, error) {
 	dir := d.dir(id)
+	if strings.Contains(id, "-init") {
+		idInitFile := d.initFile(id)
+		if _, err := os.Stat(idInitFile); err == nil {
```

Comment:
@calavera 
I don't return error because there are 2 conditions may cause 'miss parentdir file'. One is that `-init` is created by old vfs driver. The other is when build Dockerfile like this:
`FROM scratch`
`MAINTAINER dockerio`
Because scratch is a 'NoBaseImage'. So https://github.com/xuxinkun/docker/blob/vfs-driver201504/daemon/graphdriver/vfs/driver.go#L54 will return nil and will not create `parentdir`. So, I don't deal with this error.